adapters/between/betweentest/exemplary/simple-site-banner.json

```diff
@@ -0,0 +1,107 @@
+{
+  "mockBidRequest": {
```

Comment:
Could you please also add another test case that has multiple imps where a subset of imps might have the same bidder ext. This to exercise the `splitImpressions` code